src/Terminal.hh

```diff
@@ -63,6 +63,16 @@ final class Terminal implements ITerminal {
       return true;
     }
 
+    if (\getenv('TERM') === 'xterm') {
```

Comment:
Huh, I thought that this needed `xterm-\d*color`, but `TERM=xterm tput colors` agrees.

Reply:
check : https://github.com/symfony/symfony/pull/27794/files